certbot/tests/cli_test.py

```diff
@@ -1042,34 +1042,73 @@ def test_update_registration_unsafely(self):
                 "--register-unsafely-without-email".split())
             self.assertTrue("--register-unsafely-without-email" in x[0])
 
+    @mock.patch('certbot.main._determine_account')
```

Comment:
Yeah, I'll include that in a seprerate PR. The work is [here](https://github.com/cowlicks/letsencrypt/blob/gh-2716/certbot/tests/not_cli_test.py).